src/sidebar.js

```diff
@@ -343,6 +383,9 @@ Sidebar.defaultProps = {
   transitions: true,
   touch: true,
   touchHandleWidth: 20,
+  topSidebar: 0,
+  pullRight: false,
+  enableShadow: true,
```

Comment:
Could we rename this to `shadow` ? The boolean value implies the enable part of the name.